CMakeLists.txt

```diff
@@ -66,6 +66,7 @@ LIST(APPEND libsae_sources
 	sae.c
 	rekey.c
 	ampe.c
+	chan.c
```

Comment:
Is there any reason that all these sources can't be sorted?

Reply:
nope, fixed in a follow-up.